src/Traits/AssertionsTrait.php

```diff
@@ -78,7 +112,7 @@ public function assertIsRedirectResponse($response): void
      * @param string $uri
      * @param array $with
      */
-    public function assertResponseRedirectedTo($response, string $uri, array $with = [])
+    public function assertResponseRedirectedTo($response, string $uri, array $with = []): void
```

Comment:
This method should be deprecated. Laravel already offers the `assertRedirectedTo` assertion.

Reply:
Cannot do that without affecting the two methods bellow.